Build/PackageFiles/Dotnet_precompiled/ProjectTemplates-Isolated_v4.x.nuspec

```diff
@@ -16,13 +16,20 @@
     </packageTypes>
   </metadata>
   <files>
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/_gitignore" target="content/ProjectTemplate-CSharp/_gitignore" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Company.FunctionApp.csproj" target="content/ProjectTemplate-CSharp/Company.FunctionApp.csproj" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Program.cs" target="content/ProjectTemplate-CSharp/Program.cs" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/host.json" target="content/ProjectTemplate-CSharp/host.json" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/local.settings.json" target="content/ProjectTemplate-CSharp/local.settings.json" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Properties/launchSettings.json" target="content/ProjectTemplate-CSharp/Properties/launchSettings.json" />
-    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/.template.config/template.json" target="content/ProjectTemplate-CSharp/.template.config/template.json" />
+    <file src="../../../Functions.Templates/ProjectTemplate_v4.x/CSharp-Isolated/Dotnet6.0/_gitignore" target="content/ProjectTemplate-CSharp/_gitignore" />
```

Comment:
I also need to change the target here, but I'm not sure what the impact of this is. For example, can I just update this to `content/ProjectTemplate-CSharp/Net6/` or will that be an issue @soninaren